src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -288,7 +310,9 @@ internal override string GetToolName()
         /// <param name="consoleOutput"></param>
         public override void PrintHelp(TextWriter consoleOutput)
         {
-            consoleOutput.WriteLine(ErrorFacts.GetMessage(MessageID.IDS_CSCHelp, Culture));
+            consoleOutput.WriteLine(ErrorFacts.GetMessage(MessageID.IDS_CSCHelp, Culture),
+                LanguageVersion.Default.MapSpecifiedToEffectiveVersion().ToDisplayString(),
+                LanguageVersion.Latest.MapSpecifiedToEffectiveVersion().ToDisplayString());
```

Comment:
I see no value in explicitly listing the latest version, usually you get this by default, and if not, ```latest``` always works. If you want to target specific version, it is unlikely you don't know what it is, and it is probably not the latest. I propose two options:
- Either remove examples at all, including ```default``` and ```latest``` and simply say use ```?``` to get the list of supported versions, or
- hard code a couple like "6" and "7.1"